lib/generators/statesman/templates/create_migration.rb.erb

```diff
@@ -9,6 +9,8 @@ class Create<%= migration_class_name %> < ActiveRecord::Migration
       t.timestamps null: false
     end
 
+    add_foreign_key :<%= table_name %>, :<%= parent_table_name %>
```

Comment:
I think that you should change `lib/generators/statesman/templates/update_migration.rb.erb` too.